app.js

```diff
@@ -20,9 +21,10 @@ const app = express();
 app.use(express.json());
 
 app.get("/api", getEndpointsRequest);
+app.get("/api/articles", getArticlesRequest);
+// app.get("/api/articles", getArticlesByTopic);
```

Comment:
Remember to get rid of comments by the time you submit a PR! They should represent a clean, ready-to-go version of your code